integrated_channels/blackboard/client.py

```diff
@@ -38,3 +50,97 @@ def create_course_completion(self, user_id, payload):  # pylint: disable=unused-
 
     def delete_course_completion(self, user_id, payload):  # pylint: disable=unused-argument
         """TODO"""
+
+    def _create_session(self):
```

Comment:
hah yeah it's not used yet, since I did not have the transmitters etc to call. I was going to have the transmitter call the update* etc methods in my next PR where you will see this light up! I know it's odd, since this could be written now instead of new pr, but I thought it would be easier to test that change set together in transmitter pr

Reply:
and yes no one nothing else is intended to call it other than that workflow

Reply:
sounds good!